src/codecs/jpeg/encoder.rs

```diff
@@ -478,6 +493,16 @@ impl<W: Write> JpegEncoder<W> {
         }
     }
 
+    fn write_exif(&mut self) -> ImageResult<()> {
+        if !self.exif.is_empty() {
+            let mut formatted = EXIF_HEADER.to_vec();
+            formatted.copy_from_slice(&self.exif);
```

Comment:
```suggestion
            formatted.extend_from_slice(&self.exif);
```

Can we test this? In the current code, I think it is calling `<&mut [_]>::copy_from_slice` which will panic because the two lengths differ, hardly the intended effect.